build/Build.Pack.cs

```diff
@@ -527,50 +535,110 @@ string ConstructRedHatPackageFilename(string packageName, string architecture)
                         workingDirectory / $"tentacle-{framework}-{runtimeId}.{fileExtension}");
                 }
             }
-            
-            FileSystemTasks.CopyFile(ArtifactsDirectory / "zip" / $"tentacle-{FullSemVer}-{NetCoreWindows}-win-x86.zip",
-                workingDirectory / $"tentacle-{NetCoreWindows}-win-x86.zip");
-            FileSystemTasks.CopyFile(ArtifactsDirectory / "zip" / $"tentacle-{FullSemVer}-{NetCoreWindows}-win-x64.zip",
-                workingDirectory / $"tentacle-{NetCoreWindows}-win-x64.zip");
```

Comment:
Double check removing this was safe, plan is to check the actual artefacts produced against each other

Reply:
These lines are to copy the archives of the .NET 8.0 build _for Windows specifically_ (e.g. `windows-win-x64`), so that it includes the UI libs necessary for Tentacle Manager.

I've put these lines back in, and added a comment for future clarity.